### PR TITLE
Add FreezeFSForSnapshot support

### PR DESCRIPTION
**Pull request by ejweber**
longhorn/longhorn#2187

Make the volume-specific FreezeFSForSnapshot setting configurable via the UI.

Depends on https://github.com/longhorn/longhorn-manager/pull/2744.

### CODE REVIEW

File: src/routes/volume/helper/index.js
Comment (a110605) [inline src/routes/volume/helper/index.js:435]: nit: same comment. 
`option` would be changed, can declared as `const option`

**Review by a110605 — APPROVED** [review]
Overall LTGM 👍 
Does this PR need to wait for https://github.com/longhorn/longhorn-manager/pull/2744 merge first?

**Comment by ejweber** [comment]
Made the changes suggested by @a110605 and rebased. This DOES need to wait on https://github.com/longhorn/longhorn-manager/pull/2744 before merging.

**Review by a110605 — APPROVED** [review]
Thanks @ejweber, you can rebase and merge this PR once there is no dependency.

**Pull request merged**
